Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -116,12 +117,18 @@ endowed with a continuous enorm.
 We don't have `ENormedAddCommMonoid` extend `EMetricSpace`, since the canonical instance `ℝ≥0∞`
 is not an `EMetricSpace`. This is because `ℝ≥0∞` carries the order topology, which is distinct from
 the topology coming from `edist`. -/
```

Comment:
When deleting that structure, I'd like to think about what to do with that comment: but that doesn't have to be now.